card/lib/generators/card.rb

```diff
@@ -0,0 +1,43 @@
+# -*- encoding : utf-8 -*-
```

Comment:
Actually, this file is used.  It just supplies some generator classes that are used in the various generators.